### PR TITLE
DOCS-3999 Watchdog Insights for RUM Edits

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Updates the Watchdog Insights for RUM doc page and Monitoring Page Performance doc page + screenshots.

### Motivation

DOCS-3999

### Additional Notes

Merge in https://github.com/DataDog/documentation/pull/14673/ before merging this in! 

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/explorer/watchdog_insights.md [hunk 1]
```diff
@@ -1,88 +1,79 @@
...
-In the **full side panel**, you can see a timeseries of RUM errors with the field.
+In the **full side panel**, you can see a timeseries graph about the total number of RUM errors with the field along with a list of RUM events that contain the field.
 
 {{< img src="real_user_monitoring/explorer/watchdog_insights/error_outlier_side_panel.png" alt="Error Outlier full side panel" style="width:100%;" >}}
 
-### Latency outliers
+## Latency outliers
 
-Another type of insight includes latency outliers that display fields such as [faceted tags or attributes][1]) associated with performance bottlenecks and match the current query. `key:value` pairs with worse performance than the baseline provide hints into performance bottlenecks among a subset of real users.
+Latency outliers display fields such as [faceted tags or attributes][3]) that are associated with performance bottlenecks that match the current search query. `key:value` pairs with worse performance than the baseline can provide hints into the performance bottlenecks among a subset of real users.
```
Comment (alai97) [inline content/en/real_user_monitoring/explorer/watchdog_insights.md:56]:
```suggestion
Latency outliers display fields such as [faceted tags or attributes][3] that are associated with performance bottlenecks that match the current search query. `key:value` pairs with worse performance than the baseline can provide hints into the performance bottlenecks among a subset of real users.
```

File: content/en/real_user_monitoring/explorer/watchdog_insights.md [hunk 2]
```diff
@@ -1,88 +1,79 @@
...
+
 ## Further Reading
 
 {{< partial name="whats-next/whats-next.html" >}}
 
-[1]: /logs/explorer/facets/
-[2]: /real_user_monitoring/browser/monitoring_page_performance/#core-web-vitals
-[3]: /real_user_monitoring/browser/monitoring_page_performance/#monitoring-single-page-applications-spa
-[4]: /real_user_monitoring/explorer/?tab=facets#event-side-panel
+[1]: /watchdog/insights/
```
Comment (hestonhoffman) [inline content/en/real_user_monitoring/explorer/watchdog_insights.md:75]: This link 404s

File: content/en/real_user_monitoring/explorer/watchdog_insights.md [hunk 3]
```diff
@@ -1,88 +1,79 @@
...
-- link: "/real_user_monitoring/explorer/search/"
-  tag: "Documentation"
-  text: "Learn more about search in the RUM Explorer"
 - link: "https://www.datadoghq.com/blog/core-web-vitals-monitoring-datadog-rum-synthetics/#what-are-the-core-web-vitals"
   tag: "Blog"
   text: "Monitor Core Web Vitals with RUM"
 - link: "https://www.datadoghq.com/blog/datadog-mobile-rum/"
   tag: "Blog"
   text: "Improve mobile user experience with Datadog Mobile RUM"
+- link: "/watchdog/insights"
```
Comment (hestonhoffman) [inline content/en/real_user_monitoring/explorer/watchdog_insights.md:12]:
```suggestion
- link: "/watchdog/insights"
```
This link 404s (same one as below)

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 4]
```diff
@@ -13,63 +16,62 @@ further_reading:
...
-  - link: "https://www.datadoghq.com/blog/core-web-vitals-monitoring-datadog-rum-synthetics/"
-    tag: "Blog"
-    text: "Monitor Core Web Vitals with Datadog RUM and Synthetic Monitoring"
+    text: "Learn about RUM Dashboards"
 ---
 
-{{< img src="real_user_monitoring/browser/waterfall.png" alt="Real User Monitoring page load waterfall" style="width:75%;" >}}
+## Overview
+
+RUM view events collect extensive performance metrics for every single page view. Monitor your application's page views and explore performance metrics in dashboards and the RUM Explorer.
```
Comment (hestonhoffman) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:24]:
```suggestion
RUM view events collect extensive performance metrics for every page view. Monitor your application's page views and explore performance metrics in dashboards and the RUM Explorer.
```
Do you need 'single' here?

**Review by hestonhoffman — APPROVED** [review]
Broken link and a suggestion, but this is GTG once the link is fixed

File: content/en/real_user_monitoring/explorer/watchdog_insights.md [hunk 5]
```diff
@@ -1,88 +1,79 @@
...
 
-To access the full Watchdog Insights panel, click **View all**: 
+Click on an [error](#error-outliers) or [latency outlier](#latency-outliers) to interact with the visualizations embedded in the side panel and find views from the list of impacted events. Click **View all** to see all outstanding error outliers in a side panel. 
 
-{{< img src="real_user_monitoring/explorer/watchdog_insights/side_panel.png" alt="Watchdog Insights side panel" style="width:100%;" >}}
+{{< img src="real_user_monitoring/explorer/watchdog_insights/error_outlier_m_card.png" alt="Error Outlier banner card and side panel card view in the RUM Explorer" style="width:100%;" >}}
 
-Every insight comes with embedded interactions and a side panel with troubleshooting information. The insight interactions and side panel vary based on the [Watchdog Insights type](#collections).
+Hover over a card in the banner and click **Filter on Insight** to add the anomalous insight behavior in your search query. For example, you can hone in on a particular view path or a specific continent like `North America`.
```
Comment (urseberry) [inline content/en/real_user_monitoring/explorer/watchdog_insights.md:36]:
```suggestion
Hover over a card in the banner and click **Filter on Insight** to add the anomalous insight behavior to your search query. For example, you can hone in on a particular view path or a specific continent like `North America`. 
```

**Review by urseberry — APPROVED** [review]
Hi! Thanks, this looks excellent. Like Heston, I just have one tiny nit.

**Pull request merged**